client.go

```diff
@@ -50,6 +50,11 @@ const (
 	recvWindowKey = "recvWindow"
 )
 
+var (
+	// UseServerTime enables getting server time to send in the timestamp key
+	UseServerTime = false
```

Comment:
I think it is better to use RequestOption (https://github.com/adshao/go-binance/blob/master/client.go#L99) for this option